README.md

```diff
@@ -1,7 +1,6 @@
 # SCTFE
 
-This personality implements [CT Static API](https://c2sp.org/static-ct-api) using
-Trillian Tessera to store data. It is based on [Trillian's CTFE](https://github.com/google/certificate-transparency-go/tree/master/trillian/ctfe).
+This personality implements [CT Static API](https://c2sp.org/static-ct-api) using [Trillian Tessera](https://github.com/transparency-dev/trillian-tessera) to store data. It is based on [Trillian's CTFE](https://github.com/google/certificate-transparency-go/tree/master/trillian/ctfe).
```

Comment:
Avoid very long lines if possible. Though, I will admit that the standard limites are 80 or 100, and that this file doesn't respect any of these right now!

Reply:
Interesting. I thought the line width doesn't apply to the markdown files after looking at the README.md from GitHub and GOSST projects. Anyway, I've updated this line to fit into the width limit.

https://github.com/github/docs/blob/main/README.md?plain=1